luminoth/eval.py

```diff
@@ -317,7 +313,7 @@ def evaluate_once(writer, saver, ops, num_classes, checkpoint,
                 val_losses = sess.run(ops['losses'])
 
                 if image_vis is not None:
-                    filename = batch_fetched['filename'][:-4].decode('utf-8')
+                    filename = batch_fetched['filename'].decode('utf-8')
```

Comment:
Does it make sense to add this change in this PR?

Reply:
No, but let's be pragmatic